recipe/meta.yaml

```diff
@@ -1,14 +1,14 @@
 {% set name = "gufe" %}
-{% set version = "0.9.0" %}
+{% set version = "0.9.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   git_url: https://github.com/OpenFreeEnergy/{{ name }}.git
-  git_rev: 239302a
-  git_depth: 1
+  git_rev: v0.9.1
```

Comment:
Would it work if we did v{{ version }} here? One fewer thing to change on each bump.